README.md

```diff
@@ -17,4 +17,54 @@ Aside from these lines, all the other code in this repository has been written b
 
 ## Usage
 
-It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
\ No newline at end of file
+It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
+
+#Installation
+install podman from your distro's repo.
+i.e sudo dnf install podman
+Clone the files to the computer:
+git clone https://github.com/lilkidsuave/blendGuide && cd blendGuide
+Copy the following to eithier /home/(user)/.local/bin or /usr/bin
+
+blend-files
+blend-system
+blend.hook
+blend.install
+host-blend
+init-blend 
+blend-settings
+
+sudo cp blend-files blend-system blend.hook blend.install host-blend init-blend blend-settings
+
+Copy the service file to /etc/systemd/system
+sudo cp blend-files.service /etc/systemd/system
+
+Take ownership of /usr/libexec/initcripts
+sudo chown (user) -R /usr/libexec/initscripts
```

Comment:
this is not needed in manjaro gnome 43

Reply:
This was needed in Fedora 38
Your mileage may vary